todo_app/templates/index.html

```diff
@@ -9,20 +9,34 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        {% for item in items %}
-        <li> {{item.title}} </li>
+        {% for list in lists%}
+        <h3>{{ list['name']}}</h3>
+            <ul class="list-group mb-4">
+            {% for item in items if item.idList==list['idList'] %}
```

Comment:
I think you could also use a couple of extra spaces here and dot notation.
```suggestion
            {% for item in items if item.idList == list.idList %}
```